kustomize/kustomize_varsub.go

```diff
@@ -60,7 +59,7 @@ func SubstituteVariables(
 		return nil, err
 	}
 
-	key := fmt.Sprintf("%s/substitute", kustomizev1.GroupVersion.Group)
+	key := "kustomize.toolkit.fluxcd.io/substitute"
```

Comment:
I feel this should at least be moved to a `const`.

Reply:
Done!